include/oneapi/dpl/internal/random_impl/philox_engine.h

```diff
@@ -0,0 +1,406 @@
+// -*- C++ -*-
+//===-- philox-engine.h ------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Public header file provides implementation for Philox Engine
+
+#ifndef _ONEDPL_PHILOX_ENGINE_H
+#define _ONEDPL_PHILOX_ENGINE_H
```

Comment:
If I am not mistaken - the support is experimental. Please check how experimental features are handled in oneDPL.

Reply:
I've marked Philox class as experimental by moving it to `experimental` namespace. I'll need a review from oneDPL developers to double-check it was done correctly.